pkg/cache/cache.go

```diff
@@ -103,6 +103,10 @@ type Options struct {
 	// Namespace restricts the cache's ListWatch to the desired namespace
 	// Default watches all namespaces
 	Namespace string
+
+	// FieldSelectorByResource restricts the cache's ListWatch to the desired
+	// fields per resource
+	FieldSelectorByResource map[schema.GroupResource]string
```

Comment:
Please add a sample field selector here and link to the kube docs about field selectors, many ppl are not aware that they only work on a very limited set of fields and not at all for CRDs

Reply:
Done

Reply:
its `medadata.name`, not `meta.name` or not?

Reply:
Done, correct value is `metadata.name`.